ui/pages/settings/settings-tab/settings-tab.component.js

```diff
@@ -119,7 +119,16 @@ export default class SettingsTab extends PureComponent {
               id="select-currency"
               options={currencyOptions}
               selectedOption={currentCurrency}
-              onChange={(newCurrency) => setCurrentCurrency(newCurrency)}
+              onChange={(newCurrency) => {
+                setCurrentCurrency(newCurrency);
+                this.context.trackEvent({
+                  category: MetaMetricsEventCategory.Settings,
+                  event: MetaMetricsEventName.CurrentCurrencyChanged,
+                  sensitiveProperties: {
+                    current_currency_changed: newCurrency,
```

Comment:
Wondering if there is a reason why we are using `sensitiveProperties` instead of `properties` here?

Reply:
Will change! Doesn't need to be sensitive. Also needs to be added to the user object anyway 👍

Reply:
Updated! I also added the trait to `MetaMetricsUserTrait` to get added to the user, which is what we want to track (not how many times the toggle was clicked)